bolt-micronaut/src/test/java/test_locally/app/CommandsTest.java

```diff
@@ -51,7 +52,7 @@ public class CommandsTest {
     SlackSignature.Generator signatureGenerator = new SlackSignature.Generator(signingSecret);
 
     @Primary
-    @MockBean(AppConfig.class)
```

Comment:
This seemed to be the issue with the test locally. I debugged and tracked the issue down eventually and I believe there was an issue where this was being treated as a mock but the bean that was returned wasn't actually a mock, so I just made this a normal singleton and it seems to have fixed the issue. 

Partially found this as I also just ran the application locally and was able to properly instantiate the controller that was having issues in the test, main difference being the AppConfig bean so figured it had to be around that.